packages/patternfly-4/react-inline-edit-extension/src/components/InlineEdit/InlineEdit.api.test.js

```diff
@@ -1,3 +1,4 @@
+/* global beforeEach */
```

Comment:
Can we just add this to the array of globals in the eslintrc?

Reply:
Done, thanks!